src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas2/Cas2StatusUpdateTest.kt

```diff
@@ -212,41 +214,45 @@ class Cas2StatusUpdateTest(
               .expectStatus()
               .isCreated
 
-            Assertions.assertThat(realStatusUpdateRepository.count()).isEqualTo(1)
-            Assertions.assertThat(realStatusUpdateDetailRepository.count()).isEqualTo(1)
+            assertThat(realStatusUpdateRepository.count()).isEqualTo(1)
+            assertThat(realStatusUpdateDetailRepository.count()).isEqualTo(1)
 
             val persistedStatusUpdate =
               realStatusUpdateRepository.findFirstByApplicationIdOrderByCreatedAtDesc(application.id)
-            Assertions.assertThat(persistedStatusUpdate!!.assessment!!.id).isEqualTo(assessmentId)
+            assertThat(persistedStatusUpdate!!.assessment!!.id).isEqualTo(assessmentId)
 
             val persistedStatusDetailUpdate =
               realStatusUpdateDetailRepository.findFirstByStatusUpdateIdOrderByCreatedAtDesc(persistedStatusUpdate!!.id)
-            Assertions.assertThat(persistedStatusDetailUpdate).isNotNull
+            assertThat(persistedStatusDetailUpdate).isNotNull
 
             val appliedStatus = Cas2ApplicationStatusSeeding.statusList()
               .find { status ->
                 status.id == persistedStatusUpdate.statusId
               }
 
-            Assertions.assertThat(appliedStatus!!.name).isEqualTo("offerDeclined")
-            Assertions.assertThat(appliedStatus.statusDetails?.find { detail -> detail.id == persistedStatusDetailUpdate?.statusDetailId })
+            assertThat(appliedStatus!!.name).isEqualTo("offerDeclined")
+            assertThat(appliedStatus.statusDetails?.find { detail -> detail.id == persistedStatusDetailUpdate?.statusDetailId })
               .isNotNull()
 
             emailAsserter.assertEmailsRequestedCount(1)
-            emailAsserter.assertEmailRequested(
+            val email = emailAsserter.assertEmailRequested(
               toEmailAddress = applicant.email!!,
               templateId = "ef4dc5e3-b1f1-4448-a545-7a936c50fc3a",
               personalisation = mapOf(
                 "applicationStatus" to "Offer declined or withdrawn",
```

Comment:
looks like I left a stray comment in here that can be deleted - not sure what it meant by `This needs to be something else`, maybe I had meant to come back and look at the datetime stuff!

Reply:
> looks like I left a stray comment in here that can be deleted - not sure what it meant by `This needs to be something else`, maybe I had meant to come back and look at the datetime stuff!

Thanks for flagging it, i've removed it 👍